### ISSUE TITLE
Missing values description

### ISSUE THREAD

**Issue by juanfcocontreras** [man-group/dtale#164]
This is a feature request.

Would it be possible to add some basic statistics for the missing values in the column descriptions? Both the count and the percentage of the total number of values is very relevant information when doing an exploratory analysis.

Thanks!

**Comment by juanfcocontreras** [comment]
I forgot to add some code snippet.

For the whole DataFrame:
```
df.isnull().sum()  # Count
(df.isnull().sum() / len(df) * 100).round(2)  # Percentage
```

For each column:
```
df[column_name].isnull().sum()
(df[column_name].isnull().sum() / len(df) * 100).round(2)  # Percentage
```

**Comment by aschonfeld** [comment]
@juanfcocontreras I will be adding this to a release this weekend.  Is it alright if I just add the following to the list
![image](https://user-images.githubusercontent.com/11547371/80164310-9a6b2580-85a6-11ea-9f2f-59109b0f1039.png)

Just let me know if you want to change the names.

**Comment by aschonfeld** [comment]
Here's one with actual values:
![image](https://user-images.githubusercontent.com/11547371/80164360-bcfd3e80-85a6-11ea-857d-3e1f62fab70b.png)

**Comment by juanfcocontreras** [comment]
That would be great! Thank you so much!

**Comment by aschonfeld** [comment]
fixed v1.8.10
